lib/src/spec/validationEngine.spec.ts

```diff
@@ -6,168 +6,198 @@ import { FieldValidationResult } from '../entities';
 
```

Comment:
It isn't. I saw last [specs](https://github.com/Lemoncode/lcFormValidation/pull/32/files#diff-0cca4b84b6f407ccfec363df7bfcf4c8R172)